spec/dmv_spec.rb

```diff
@@ -3,9 +3,9 @@
 RSpec.describe Dmv do
   before(:each) do
     @dmv = Dmv.new
-    @facility_1 = Facility.new({name: 'DMV Tremont Branch', address: '2855 Tremont Place Suite 118 Denver CO 80205', phone: '(720) 865-4600'})
-    @facility_2 = Facility.new({name: 'DMV Northeast Branch', address: '4685 Peoria Street Suite 101 Denver CO 80239', phone: '(720) 865-4600'})
-    @facility_3 = Facility.new({name: 'DMV Northwest Branch', address: '3698 W. 44th Avenue Denver CO 80211', phone: '(720) 865-4600'})
+    @facility_1 = Facility.new({:name => 'DMV Tremont Branch', :address => '2855 Tremont Place Suite 118 Denver CO 80205', :phone => '(720) 865-4600'})
+    @facility_2 = Facility.new({:name => 'DMV Northeast Branch', :address => '4685 Peoria Street Suite 101 Denver CO 80239', :phone => '(720) 865-4600'})
+    @facility_3 = Facility.new({:name => 'DMV Northwest Branch', :address => '3698 W. 44th Avenue Denver CO 80211', :phone => '(720) 865-4600'})
```

Comment:
The original key/value syntax (`{name: 'DMV Tremont Branch', address: '2285...'}`) is more conventional in newer versions of Ruby, but both will work.